README.md

```diff
@@ -1,66 +1,78 @@
+# Introduction
```

Comment:
Can you swap the final two commits, that is update the documentation, then bump the version. Seems as if the documentation is not part of the change otherwise.

Reply:
Certainly.